docs/developer/release/1-create-release-branch.md

```diff
@@ -0,0 +1,37 @@
+# Create Release Branch
+
+A single release branch is created for every major or minor release. That release
+branch is then used for all Release Candidates, the Stable Release, and all
+Patch Releases.
+
+## Release Version Prefix
+
+The `[release version prefix]` can be determined by looking at the last release
+version and adding to it. Also, drop off the 3rd number as shown in the examples below.
```

Comment:
Spell out ordinals less than 10, unless they're used for math:

```suggestion
version and adding 1 to it. Also, drop off the third number as shown in the examples below:
```

From Google's developer guide on [Numbers](https://developers.google.com/style/numbers).